ecf/include/envir-p1.h

```
@@ -0,0 +1,46 @@
+# envir-p1.h
+export job=${job:-$PBS_JOBNAME}
+export jobid=${jobid:-$job.$PBS_JOBID}
+
+export RUN_ENVIR=emc
+export envir=%ENVIR%
+export MACHINE_SITE=%MACHINE_SITE%
+export SENDDBN=${SENDDBN:-%SENDDBN:YES%}
+export SENDDBN_NTC=${SENDDBN_NTC:-%SENDDBN_NTC:YES%}
+#module load prod_envir/2.0.4 prod_util/2.0.9 envvar/1.0
+echo debug
+if [[ "$envir" == prod && "$SENDDBN" == YES ]]; then
+    export eval=%EVAL:NO%
+    if [ $eval == YES ]; then 
+      export SIPHONROOT=${UTILROOT}/para_dbn
+    else 
+      export SIPHONROOT=/lfs/h1/ops/prod/dbnet_siphon
+    fi
+#### Developer overwrite
+    export SIPHONROOT=${UTILROOT}/fakedbn
+####
+else
+    export SIPHONROOT=${UTILROOT}/fakedbn
+fi
+
+export DBNROOT=$SIPHONROOT
+
+####if [[ ! " prod para test " =~ " ${envir} " && " ops.prod ops.para " =~ " $(whoami) " ]]; then err_exit "ENVIR must be prod, para, or test [envir-p1.h]"; fi
+export ECF_PORT=34326
```

Please remove this line.  This variable is set when `ecflow` module is loaded.  `ecflow` module is loaded in `head.h`. This variable is unique to the user.

---

The job tested without this line sometime failed silently. There is a good reason to include this in here.
As discussed, this file will be used only for reference. I prefer to keep it.

---

Sometimes failure is a symptom of the system instability.  I would rather encounter this failure sometimes, than hard-wire some users personal ECF_PORT information.  I suggest removal.

---

Removed